addon/components/-private/animated-popper.js

```diff
@@ -190,6 +190,10 @@ export default class AnimatedPopperComponent extends Component {
    */
   finalizeClose() {
     run(() => {
+      if (this.get('isDestroyed')) {
```

Comment:
`this.isDestroyed` is sufficient, this is just a property and not a CP or anything.

And this check could also be performed before the `run(() => {`!

Reply:
oh cool stuff, I've always used this.get('isDestroyed')
and alright I'll put it before